src/components/RuleBook/index.js

```diff
@@ -1,9 +1,10 @@
 import React from "react";
 import styled from "styled-components";
+import PropTypes from "prop-types";
 
-export default function RuleBook() {
+export default function RuleBook({ onClick }) {
   return (
-    <ModalDiv>
+    <ModalDiv onClick={onClick}>
```

Comment:
props 네이밍에 대해서 고민해보면 좋을 것 같습니다. 
```suggestion
    <ModalDiv onClick={handleClick}>
```